core/scripts/local/VoteGasEstimation.js

```diff
@@ -52,60 +53,104 @@ async function run() {
   // Pick a random time. Probabilistically, this won't request a duplicate time from a previous run.
   let time = Math.floor(Math.random() * max);
 
-  console.log("total supply", (await votingToken.totalSupply()).toString());
+  console.log("Voting tokens total supply:", (await votingToken.totalSupply()).toString());
 
   for (var i = 0; i < numRounds; i++) {
-    console.log("Round", i);
+    console.group(`\n*** Round ${i} ***`);
     await cycleRound(voting, votingToken, identifier, time, accounts);
     time += numPriceRequests;
+    console.groupEnd();
   }
 
+  console.group(`\nVoter token balances post-test:`);
   for (var i = 0; i < numVoters; i++) {
     const voter = getVoter(accounts, i);
-    console.log("balance", (await votingToken.balanceOf(voter)).toString());
+    const balance = await votingToken.balanceOf(voter);
+    console.log(`- Voter #${i}: ${balance.toString()}`);
   }
-  console.log("total supply", (await votingToken.totalSupply()).toString());
+  console.groupEnd();
+  console.log("Voting tokens total supply", (await votingToken.totalSupply()).toString());
 
-  console.log("Done");
+  console.groupEnd();
 }
 
+// Cycle through each round--consisting of a price request, commit, and reveal phase
 const cycleRound = async (voting, votingToken, identifier, time, accounts) => {
+  /**
+   * Estimating gas usage: requestPrice
+   */
+  let gasUsedPriceRequest = 0;
+  console.group(`\nEstimating gas usage: requestPrice`);
+
   for (var i = 0; i < numPriceRequests; i++) {
     const result = await voting.requestPrice(identifier, time + i, { from: accounts[1] });
-    console.log("requestPrice", result.receipt.gasUsed);
+    const _gasUsed = result.receipt.gasUsed;
+    console.log(`Price Request #${i}: ${_gasUsed}`);
+    gasUsedPriceRequest += _gasUsed;
   }
+  console.log(`Total gas: ${gasUsedPriceRequest}`);
+  console.groupEnd();
 
+  // Advance to commit phase
   await moveToNextRound(voting);
 
   const salts = {};
   const price = getRandomSignedInt();
+
+  /**
+   * Estimating gas usage: commitVote
+   */
+  let gasUsedCommitVote = 0;
+  console.group(`\nEstimating gas usage: commitVote`);
+
   for (var i = 0; i < numPriceRequests; i++) {
+    console.group(`Price request #${i}`);
+
     for (var j = 0; j < numVoters; j++) {
       const salt = getRandomUnsignedInt();
       const hash = web3.utils.soliditySha3(price, salt);
 
       const voter = getVoter(accounts, j);
 
       const result = await voting.commitVote(identifier, time + i, hash, { from: voter });
-      console.log("commitVote", result.receipt.gasUsed);
+      const _gasUsed = result.receipt.gasUsed;
```

Comment:
Similar comment as the other script, is there any way we can get the gas *required* to run the txn, too?